addons/crm/wizard/crm_update_probabilities.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class CrmUpdateProbabilities(models.TransientModel):
+    _name = 'crm.update.probabilities'
+    _description = "Update the probabilities"
+
+    def _get_default_pls_start_date(self):
+        pls_start_date_config = self.env['ir.config_parameter'].sudo().get_param('crm.pls_start_date')
+        return fields.Date.to_date(pls_start_date_config)
```

Comment:
If we don't have any what happens ?

Reply:
In a worst case scenario, if someone removes existing required attribute from the settings view, the date in the modal body will be blank. Nothing breaks. At the end, [`_update_automated_probabilities`](https://github.com/odoo/odoo/blob/74ea66a107cfc98cce8e52903ec21e3d752fb4fb/addons/crm/models/crm_lead.py#L1666) method always makes sure that probabilities are not updated if config parameter is not found. However, I've added a separate string in such case to make it understandable for users. If someone removes required attribute from the settings view and theres no config param, modal will ask user to select the date first and save the settings (and there will be only `Cancel` button on the modal in this case). Let me know if that works for you and LNA.